ethereum/viper
# Move parser integration tests into an integration folder

### - What I did
I broke the integration tests in `test_parser.py` into separate folders/files.
### - How I did it
I went through all the integration tests in `test_parser.py` and figured out where they belong.

### - How to verify it
Pull down my branch and run all the tests / go through my PR to make sure where I moved the integration tests makes sense.

### - Description for the changelog
None
### - Cute Animal Picture
![image](https://user-images.githubusercontent.com/17552858/30551336-c776e254-9c5f-11e7-95fa-39a3ee2dcb02.png)
